CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 * Upgrade to Goji 2.0.2 and use the modern Goji mux API. Thanks [hans-stripe](https://github.com/hans-stripe)!
 * Increase the timeout of TestCountActiveHandlers from 3 seconds to 10 seconds. Helps some tests pass in certain environments. Thanks [sushain97](https://github.com/sushain97)!
 
+## Bugfixes
+* A fix for forwarding metrics with gRPC using the kubernetes discoverer. Thanks, [androohan](https://github.com/androohan)!
+
```

Comment:
Can you create a new 14.2.0 section at the top of the readme and move this there?

Reply:
Updated, wasn't sure to put for the date so put unreleased